docs/configuring-playbook-postgres-backup.md

```diff
@@ -0,0 +1,42 @@
+# Setting up postgres backup (optional)
+
+The playbook can install and configure [docker-postgres-backup-local](https://github.com/prodrigestivill/docker-postgres-backup-local) for you.
+
+## Adjusting the playbook configuration
+
+| Name                              | Default value                | Description                                                      |
+| :-------------------------------- | :--------------------------- | :--------------------------------------------------------------- |
+|matrix_postgres_backaup_enabled|false|Set to true to use [docker-postgres-backup-local](https://github.com/prodrigestivill/docker-postgres-backup-local) to create automatic database backups|
+|matrix_postgres_backup_schedule| '@daily' |Cron-schedule specifying the interval between postgres backups.|
+|matrix_postgres_backup_keep_days|"7"|Number of daily backups to keep|
+|matrix_postgres_backup_keep_weeks|"4"|Number of weekly backups to keep|
+|matrix_postgres_backup_keep_months|"12"|Number of monthly backups to keep|
+|matrix_postgres_backup_path | "{{ matrix_base_data_path }}/postgres-backup" | Storagepath for the database backups|
+|matrix_postgres_backup_matrix_reminder_bot_enabled| false | Set to true to backup matrix_reminder_bot database. Per default matrix-reminder-bot uses an sqlite database. Only use this if you are using a postgres database for matrix-reminder-bot. |
```

Comment:
Do we need the flexibility of being able to turn each of this on and off manually?

Wouldn't it be better if we back up all services by default?

Reply:
I added an configuration option for each of them because the used container prodrigestivill/docker-postgres-backup-local requires a defined list of databases to backup. I did not know a solution to detect the used postgresdatabases during runtime thus i created the configuration options. Do you know a way to detect the used postgres databases?

Reply:
I reworked this to use the `matrix_postgres_additional_databases` variable in `group_vars/matrix_servers`